### PR TITLE
AST-44 - refactor: Block Editor & GB layout improvements

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Spacing & alignment related issues in GB blocks & editor. 
- Issue with the left-right spacing in Gutenberg editor for pattern & normal blocks
- Added design styelsheet to current page number in pagination
- In previous view there is no such CSS for active paginated number (refer following last point in Screenshots)

### Screenshots
- Inner spacing issue - Astra - [SS](https://share.getcloudapp.com/jkuepBvD) | TwentryTwentyOne - [SS](https://share.getcloudapp.com/mXu1W42g) -> [FIXED](https://a.cl.ly/6qu8K79K)
- Cover / Group block - https://share.getcloudapp.com/4gu1vd5j
- Group block - https://share.getcloudapp.com/qGuX8kvq 
- Group block - https://share.getcloudapp.com/NQuJm8Q7
- Group block - https://share.getcloudapp.com/nOulrqlkl
- Design added for current page in pagination - [Previous](https://a.cl.ly/Jru4kA4x) -> [New](https://a.cl.ly/12uAxBZN)

### Types of changes
- Non-breaking CSS change
- As managed with backward conditions & with filter

### How has this been tested?
- With creating new GB pages & test in block editor as well
- Check with alignleft / alignright block layout

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by Nikschavan** [comment]
Task linked: [CU-jwk5e6 Block Editor compatibility CSS](https://app.clickup.com/t/jwk5e6)

File: inc/core/class-gutenberg-editor-css.php [hunk 1]
```diff
@@ -496,6 +493,35 @@ public static function get_css() {
 				);
 
 				$css .= astra_parse_css( $button_patterns_mobile_compat_css, '', astra_get_mobile_breakpoint() );
+
+				$gb_patterns_min_mobile_css = array(
```
Comment (rohitp-bsf) [inline inc/core/class-gutenberg-editor-css.php:497]: Can we get this in `if..else`?

File: inc/class-astra-dynamic-css.php [hunk 2]
```diff
@@ -1063,6 +1063,35 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 				);
 
 				$parse_css .= astra_parse_css( $outline_button_mobile_css, '', astra_get_mobile_breakpoint() );
+
+				$gb_patterns_min_mobile_css = array(
```
Comment (rohitp-bsf) [inline inc/class-astra-dynamic-css.php:1067]: Can we get this in `if..else`?

**Review by rohitp-bsf — CHANGES_REQUESTED** [review]

File: inc/core/class-gutenberg-editor-css.php (hunk 1 above)
Comment (imnavanath) [reply inc/core/class-gutenberg-editor-css.php]: Done 👍

File: inc/class-astra-dynamic-css.php (hunk 2 above)
Comment (imnavanath) [reply inc/class-astra-dynamic-css.php]: Done 👍

**Review by rohitp-bsf — DISMISSED** [review]

**Review by vrundakansara — APPROVED** [review]

**Review by rohitp-bsf — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

**Pull request merged**